package.json

```diff
@@ -54,6 +58,9 @@
     "babel-jest": "^26.1.0",
     "babel-loader": "^8.1.0",
     "browserslist-config-openmrs": "^1.0.1",
+    "carbon-components": "^10.21.0",
+    "carbon-components-react": "^7.21.0",
```

Comment:
This is not a shared dependency.

Could you move it to `dependencies`?

Reply:
(Explanation: Right now we will share `carbon-components` and `carbon-icons` from the app shell. The React ones we will potentially not share as the shell should mostly remain framework neutral where possible.)

(The comment thus is only targeted at `carbon-components-react`)

Reply:
Maybe its just me, but `carbon-components-react` is showing up in `devDependencies` and `dependencies`. Remove it from the `devDependencies` please.